src/js/01-nav.js

```diff
@@ -616,102 +616,114 @@
-      const navVersionDropdown = createElement('.nav-version-dropdown')
-      navVersionDropdown.addEventListener('click', trapEvent)
-      const navVersionButton = createElement('button.button.nav-version-button')
-      navVersionButton.setAttribute('tabindex', '-1')
       const activeVersion = componentData.name === page.component ? page.version : currentVersionData.version
-      const activeDisplayVersion = componentData.versions[activeVersion].displayVersion
-      const navVersionLabel = createElement('label', activeDisplayVersion)
-      navVersionLabel.id = `combo-${componentData.name}-label`
-      navVersionButton.appendChild(navVersionLabel)
       const navVersion = createElement('div.nav-version', { dataset: { version: activeVersion } })
-      navVersion.setAttribute('tabindex', '0')
-      navVersion.setAttribute('role', 'combobox')
-      navVersion.ariaExpanded = false
-      navVersion.ariaHasPopup = 'listbox'
-      navVersion.id = `combo-${componentData.name}`
-      navVersion.setAttribute('aria-labelledby', `combo-${componentData.name}-label`)
-      navVersion.setAttribute('aria-controls', `listbox-${componentData.name}`)
-      if (activeVersion === currentVersionData.version) {
-        addCurrentVersionIndicator(navVersionButton, 'tooltip-dot-nav-version-menu')
-      }
-      navVersionButton.appendChild(navVersion)
-      if (page.navVersionIconId) {
-        navVersionButton.appendChild(createSvgElement('.icon.nav-version-icon', '#' + page.navVersionIconId))
-      }
+      const activeDisplayVersion = componentData.versions[activeVersion].displayVersion
+
       const navVersionMenu = createElement('div.nav-version-menu')
       navVersionMenu.setAttribute('role', 'listbox')
       navVersionMenu.id = `listbox-${componentData.name}`
       navVersionMenu.setAttribute('aria-labelledby', `combo-${componentData.name}-label`)
-      versions.reduce((lastVersionData, versionData) => {
-        if (!isArchiveSite()) {
-          if (versionData === currentVersionData) {
-            navVersionMenu.appendChild(createElement('span.nav-version-label', currentVersion))
-          } else if (versionData.prerelease) {
-            if (!lastVersionData) {
-              navVersionMenu.appendChild(createElement('span.nav-version-label', 'Prerelease versions'))
-            }
-          } else if (lastVersionData === currentVersionData) {
-            navVersionMenu.appendChild(createElement('span.nav-version-label', previousVersions))
-          }
-        } else if (versionData === currentVersionData) {
-          navVersionMenu.appendChild(createElement('span.nav-version-label', 'Archived versions'))
+
+      let navVersionButton
+
+      if (versions.length === 1) {
+        navVersionButton = createElement('p.prev-flag')
+        navVersionButton.textContent = activeDisplayVersion
+        navVersionButton.title = `${activeDisplayVersion} is the current and only version of ${componentData.title}`
```

Comment:
Sorry, missed this the first time around, but just noticed this is new text we'll need to localize.

No need to take care of it now necessarily, but would you mind adding it to the backlog? It might be easier to keep the message generic as well since interpolation doesn't work well with translations. Maybe just: "This is the current and only version."

Reply:
sounds good. I will create a GUS ticket for localizing it and to make it more generic.

Reply:
GUS ticket created: [W-15145820](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001l7X6SYAU/view)